src/core/models/navigation.js

```diff
@@ -16,4 +16,14 @@ export default class Navigation {
     }
     return new Navigation(nav);
   }
+
+  /**
+   * Constructs a Navigation component from an answers-core VerticalResults array
```

Comment:
nit: This constructs the model, not component, right?

Reply:
ya, i'll update